src/InstrumentSchema.php

```diff
@@ -116,25 +116,14 @@ public static function wrap_field_resolvers( $fields, $type_name ) {
 							'startOffset' => $start_offset,
 						];
 
-						try {
-
-							/**
-							 * If the current field doesn't have a resolve function, use the defaultFieldResolver,
-							 * otherwise use the $field_resolver
-							 */
-							if ( null === $field_resolver || ! is_callable( $field_resolver ) ) {
-								$result = Executor::defaultFieldResolver( $source, $args, $context, $info );
-							} else {
-								$result = call_user_func( $field_resolver, $source, $args, $context, $info );
-							}
-
-						} catch ( \Exception $error ) {
-
-							/**
-							 * Throw an exception for the error that was returned from the resolver
-							 * @since 0.0.1
-							 */
-							throw new \Exception( $error );
```

Comment:
@jasonbahl, did you remove this because it shouldn't throw an error?

Reply:
@hughdevore I removed the `try/catch` because it conflicts with how GraphQL-PHP resolves errors in the newer version. 

See: https://github.com/webonyx/graphql-php/issues/188#issuecomment-337301434

Reply:
Errors still will get logged, just at a higher level than here

Reply:
@jasonbahl, sounds good, thanks! I think you have the admin privilege to ignore the tests and merge this? If so, feel free to.